db/seeds.rb

```diff
@@ -27,7 +33,16 @@
   body: "A Contact List will be implemented here eventually."
 )
 
-puts "Seed finished"
-puts "#{about_page.menu_title} page was created"
-puts "#{upcoming_events.menu_title} page was created"
-puts "#{contacts.menu_title} page was created"
+photos = Page.create!(
+  menu_title: "Photos",
+  title: "Photos",
+  body: "A list of photos will be implemented here eventually!"
+)
+
+videos = Page.create!(
+  menu_title: "Videos",
+  title: "Video Gallery",
+  body: "Here we shall have a video gallery showcasing Lorbo's supreme supremeness."
+)
+
+puts "Seeds finished"
```

Comment:
See how this commit actually has two concerns mixed in, lessening project's readability? That's when `git staged` comes in handy right before `git commit`: had we have checked, we would have split the commit in two I guess :)